### DISCUSSION TITLE
Resize Scroll bar

### DISCUSSION THREAD

**Discussion by roderixgit** [GetPublii/Publii#1792] [Ideas]
First of all, congratulations for the great job and wonderful tool.

I would like to report an issue that annoys me so much.
The scroll bar in Windows desktop is too arrow, and the cursor too small. It is hardly, not to say impossible, to use a standard mouse to scroll down to the end of the post.
The use of the keyboard seems to be the only solution, but it's exhaustive for long posts/pages.

My suggestion is to resize the scroll bar in a way that a standard mouse can be used. The Windows default would be perfect.

Cheers,
Rod

**Comment by bobmitro** [comment]
Abracadabra, make it so fine,
Let Windows' scrollbar be wider this time :)

Just enable the "Use wider scrollbars - https://getpublii.com/docs/publii-global-settings.html

**Reply by roderixgit** [reply]
WOW

Thank you so much. I have lived with this dilemma for so long. 
Yes, this is it!.
